### ISSUE TITLE
animorph masks

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#15928]
(WEB REPORT BY: tornader REMOTE: 172.93.109.202:7777)
# Revision
0b8e518ccddb7287a8f1854f42cf0cc0a08d0efa

# Description
the ayy lmao (xeno) mask turns you invisible and you still get access to the xeno abilities

# Steps to Reproduce
use xeno mask

# What you Expected

# What Actually Happened

**Comment by Shadowmech88** [comment]
Likely due to that specific type of xeno not having a sprite because it's not designed to be spawned.

**Comment by Hugh-G-Rection** [comment]
While this is open figured I'd add some other issues I've found with these things.

Swarm of Bees: The bees will swarm each other, killing the mob and ghosting you

Giant Spider: Apparently the creation of the giant spider triggers the "make queen" check. Was put on three times, on the third time the giant spider turned into a Spiderqueen, forcing out the player into ghost and giving the queen it's mob AI

SCP-173: This is probably the funniest. 
![r i p](https://user-images.githubusercontent.com/27928368/30841127-6b814f84-a240-11e7-96a1-f479a3fc61d0.png)
That's right, you snap your own neck.
I believe it's only if you put it on and take it off rapidly, the mob exists separately from you for a quarter of a second, just enough time for CRUNCH.

I hesitate to bring these up, just because the mask is one of my favorite additions, and don't want any of them removed, especially SCP.

**Comment by Shadowmech88** [comment]
Actually, it's possible that SCP-173 lacks sanity to the degree that it's able to snap the neck of a mob which is inside its own contents. The user's death would then cause the mask to come off and SCP-173 to be deleted.

**Comment by Hugh-G-Rection** [comment]
I know that can't be the case in all scenarios, because later that same round someone put on the mask and walked around for a while, accidentally snapping the necks of two people he walked by in the hallway before taking it off. The only differences noticed was that he had the mask on for longer before taking it off, and when taking the mask off he was buckled to a rolling chair which may have prevented the self snap.
The only other difference obviously is that he broke someone else's neck before taking it off successfully, which if I could just head-canon this bug for a second, it would be fantastic if you were forced to sate the mobs snap-lust before you were allowed to safely remove the mask.

At any rate these things are an under-utilized goldmine of hilarity that I'd love to see more of before someone who hates fun removes them for "muh spess balance"

**Comment by Shadowmech88** [comment]
In testing locally, the spawned SCP-173 snapped my neck as soon as I put the mask on, every time. Interestingly however, since putting a morphing mask on places your original body in suspended animation, I was able to live until I took the mask off. Also strange is the fact that when I placed the mask on a monkeyman, the SCP didn't snap its neck.

However, I'm not certain that these issues should be fixed. SCP-173 is an invulnerable mob, so even in spite of the incredible rarity of receiving a mask of it, I think allowing it to function without at least severe danger to the wearer would make it too powerful.
